bootstrap/018_warehouse_updates.sql

```diff
@@ -11,6 +11,7 @@ BEGIN
     let migrate2 string := null;
     if (migrate) then
         SYSTEM$LOG_TRACE('Migrating data.');
+        call internal.migrate_view();
```

Comment:
the ordering of this looks strange to me. And doesnt it mean that all views will get updated when only the wrehouse tables will get updated in this case?

Reply:
for this `018_warehouse_updates.sql`, we do not have to call `internal.migrate_view()`, as the view in `018_warehouse_updates.sql` do not uses `query_history` directly.  Probabaly I should have remove this line of code change.

Reply:
changed `018_warehouse_updates.sql` such tat we do not call `internal.migrate_view()`.